PersonBook/PersonBook/PersonBook/Program.cs

```diff
@@ -1,78 +1,88 @@
-               "~~~~~~Welcome to Person Book Application~~~~~\n " +
-               "Please Type\n " +
-               "1- List Persons\n " +
-               "2- Add new Person\n " +
-               "3- Update Person\n " +
-               "4- Delete Person\n " +
-               "5- exit\n");
+                PrintOptions();
                 string option = Console.ReadLine();
-              
+
                 switch (option)
                 {
-                    case  "1":
-
-                            pservice.ListPersons().ForEach(p => {
-                                Console.WriteLine("********************************\n");
-                                Console.WriteLine("Id : " + p.Id + "\n" +
-                                    "First Name : " + p.FirstName + "\n" +
-                                    "Last Name : " + p.LastName);
-                                Console.WriteLine("********************************\n");
-                            });
+                    case "1":
+                        foreach (Person item in personService.ListPersons())
+                        {
+                            ShowPerson(item);
+                        }
                         break;
 
                     case "2":
 
-                        Console.WriteLine("Enter name and surname");
+                        Console.WriteLine("Name: \nSurname : ");
                         string name = Console.ReadLine();
-                        string surname = Console.ReadLine();
-                        Person newPerson =new Person(name,surname);
-                        pservice.AddPerson(newPerson);
+                        string surname = Console.ReadLine();                     
+                        personService.AddPerson(name,surname);
                         break;
 
                     case "3":
 
-                        Console.WriteLine("Please enter an ID and new Name, Surname ");
-                        int inputId = int.Parse(Console.ReadLine());
+                        Console.WriteLine("Id : \nName: \nSurname : ");
+                        int inputId;
+                        Int32.TryParse(Console.ReadLine(), out inputId);
                         string inputName = Console.ReadLine();
                         string inputSurname = Console.ReadLine();
-                        Person pUpdate = new Person(inputId, inputName, inputSurname);
-                        pservice.UpdatePerson(pUpdate);
+                        personService.UpdatePerson(inputId,inputName,inputSurname);
                         break;
 
                     case "4":
-
-                        Console.WriteLine("Please enter an ID ");
-                        int deleteId = int.Parse(Console.ReadLine());
-                        pservice.DeletePerson(deleteId);
+                        Console.WriteLine("Id: ");
+                        int number;
+                        Int32.TryParse(Console.ReadLine(),out number);
```

Comment:
you don't need to define `int number`. you can do it like below.
`Int32.TryParse(Console.ReadLine(),out var number)`